pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -535,26 +535,27 @@ func (sf *streamIngestionFrontier) maybePersistFrontierEntries() error {
 }
 
 func (sf *streamIngestionFrontier) maybeCheckForLaggingNodes() error {
-	defer func() {
-		sf.lastNodeLagCheck = timeutil.Now()
-	}()
+	ctx := sf.Ctx()
+	log.VEventf(ctx, 2, "consider lag replanning check")
```

Comment:
I'd probably remove this one since you log before every return.

Reply:
Done.